Tests/TuistKitTests/Services/GraphServiceTests.swift

```diff
@@ -64,4 +65,34 @@ final class GraphServiceTests: TuistUnitTestCase {
         Graph exported to \(graphPath.pathString)
         """)
     }
+    
+    func test_run_whenJson() throws {
```

Comment:
Checking the result is very simple here, why don't we do it? It would help to catch future regressions in the export

Reply:
What do you mean? It is checking the results by deserialize it. Marek has suggested to do this.